aclk/schema-wrappers/schema_wrapper_utils.cc

```diff
@@ -13,3 +13,10 @@ void set_timeval_from_google_timestamp(const google::protobuf::Timestamp &ts, st
     tv->tv_sec = ts.seconds();
     tv->tv_usec = ts.nanos()/1000;
 }
+
+int label_add_to_map_callback(const char *name, const char *value, RRDLABEL_SRC ls, void *data) {
+    (void)ls;
```

Comment:
A minor suggestion, we have defined a macro [UNUSED](https://github.com/netdata/netdata/blob/e61633a9e99e97cf325514b96f08ccafdb640674/libnetdata/libnetdata.h#L301) that inserts a code like this, I suggest to use it.

Reply:
I do not remember who brought it for us, but since it was merged, we have used it hundred of times:

```sh
bash-5.1$ grep -r -w "UNUSED" *  | wc -l
205
```

My suggestion was in the direction to keep an unique pattern, but  I understand your issues. Let us keep your PR as it is, and we can think about other PR to return for the explicit method.  :)